main.go

```diff
@@ -65,17 +80,29 @@ func parseConfig() (*config, error) {
 			return nil, errors.AddContext(err, "failed to parse log level")
 		}
 	}
-	cfg.DBURI, ok = os.LookupEnv(mongoDBURI)
+	cfg.DBURI, ok = os.LookupEnv(envMongoDBURI)
+	if !ok {
+		return nil, fmt.Errorf("%s wasn't specified", envMongoDBURI)
+	}
+	cfg.DBUser, ok = os.LookupEnv(envMongoDBUser)
+	if !ok {
+		return nil, fmt.Errorf("%s wasn't specified", envMongoDBUser)
+	}
+	cfg.DBPassword, ok = os.LookupEnv(envMongoDBPassword)
+	if !ok {
+		return nil, fmt.Errorf("%s wasn't specified", envMongoDBPassword)
+	}
+	cfg.SkydOpts.Address, ok = os.LookupEnv(envSkydAPIAddr)
 	if !ok {
-		return nil, fmt.Errorf("%s wasn't specified", mongoDBURI)
+		return nil, fmt.Errorf("%s wasn't specified", envSkydAPIAddr)
 	}
-	cfg.DBUser, ok = os.LookupEnv(mongoDBUser)
+	cfg.SkydOpts.UserAgent, ok = os.LookupEnv(envSkydAPIUserAgent)
 	if !ok {
-		return nil, fmt.Errorf("%s wasn't specified", mongoDBUser)
+		return nil, fmt.Errorf("%s wasn't specified", envSkydAPIUserAgent)
 	}
```

Comment:
Do we need to force the user to configure this? We can default to `Sia-Agent` instead.

Reply:
I like being explicit usually because it's a tiny effort when setting up but it gives you very strong guarantees of not accidentally breaking but I guess for the agent it's fine. There is really no point in ever using anything besides the default.